WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,227 @@
+
+Generally, WEPs do not contain the names of individual people, but rather references to teams in the Write the Docs organization. Similarly, they don’t contain fixed budget numbers, but instead guidelines for how to implement the budget. 
+
+How to submit a WEP
+-------------------
+
+Anyone from the community can submit a WEP! Here's how it works, and what to expect.
+
+The WEP process and system are administered by the following teams:
+
+Core Operations team
+  There are several reference in this WEP to the **core operations (CoreOps) team**. This team is currently known as the **core team** and the team description can be found on the `Core team page <https://www.writethedocs.org/team/#core-team>`_. The updated name is intended to reflect the purpose of the team more accurately in the context of its duties for the community. 
+
+WEP team
+  The **WEP team** is a new team that reviews WEP proposals for clarity and completeness. For the purpose of this WEP the team is initially composed of the Core Operations team, but in the the future will be open for other community members to join. 
+  
+Note that the process to join and leave teams is out of scope of this WEP and will be handled in a future WEP.
+
+At a very high level, the WEP submission process includes the following stages:
+
+#. `Pre-proposal`_ — Someone has an idea and they start collecting early input and feedback to see if it is within the scope of a WEP. This can be done informally, publicly or privately.
+
+#. `Draft submission and pre-review`_ — The WEP author writes a rough draft of the WEP and submits it in a pull request (PR) to the WEPs repository. The draft is pre-reviewed by an editor for style, formatting, and eligibility for the WEP process. If the WEP is pre-approved, it is accepted for discussion and the 30-day review period begins.
+
+#. `Community review, discussion, and updates`_ — During the review period, the WEP draft is discussed, improved, and updated based on incoming feedback from the community. Feedback is welcome from the whole community, as long as it arrives within the review period. 
+
+#. `Final review and resolution`_ — At the end of the review period, the WEP is examined for **lazy consensus**. A lazy consensus means a situation where the community either provided feedback, approved the WEP explicitly, or ignored the WEP altogether. In some cases where consensus wasn't reached, the review period might be extended, but never shortened.
+
+#. `Implementation`_ — If the WEP is accepted, the processes or policies described in the WEP are implemented by the CoreOps team, the WEP author, or specific community members as defined in the WEP.
+
+The following sections describe each stage in more detail.
+
+Pre-proposal
+............
+
+The WEP process begins with a new idea for Write the Docs. It is highly recommended that a single WEP contain a single key proposal or new idea. 
```

Comment:
Does it have to be a "new idea"?

If the WEP contains multiple proposals, I think that would come out in the discussion.

Reply:
I don't think we're going to judge things by any new-ness quality, it's more just a turn of phrase. Perhaps "process update" is more explicit?

Reply:
In the tech writing world, the word "new" is usually redundant. I vote we clean it up and just say "idea".